SofaKernel/framework/sofa/core/BaseMapping.cpp

```diff
@@ -44,14 +44,14 @@ BaseMapping::~BaseMapping()
 
 bool BaseMapping::setFrom(BaseState*  )
 {
-    this->serr<<"BaseMapping::setFrom is not implemented for " << this->getName()<< sendl;
+    dmsg_error() << "Calling a virtual method not implemented." ;
```

Comment:
@damienmarchal Could you please remind me the difference between msg_error and dmsg_error and where it is specified in the code?

Reply:
Of course, 
all the details are there: https://www.sofa-framework.org/community/doc/programming-with-sofa/start-coding/message-api/